src/blog/2019-04-03/one-mutation-testing-html-report.pug

```diff
@@ -0,0 +1,98 @@
+        
+        Mutation testing elements comes with batteries included:
+        
+        📊 Calculates and displays the mutation score and other metrics  
+        📁 Group results into directories  
+        👓 Show mutants directly in your source code  
+        😎 Code highlighting with [highlightjs](https://highlightjs.org/)  
+        🧙‍ Filter mutants based on the outcome  
+        🔗 Deep linking using anchors (uses fragment, so path will not be contaminated)  
+        ✨ A nice (yet basic) UI with embedded [bootstrap](https://getbootstrap.com) css  
+        🎓 Adheres to [custom element best practices](https://developers.google.com/web/fundamentals/web-components/best-practices)
+
+        ## 💭 How it works
+
+        Using mutation testing elements is as simple as loading the JavaScript bundle and adding 
+        the `mutation-test-report-app` to your page, pointing the `src` to a json file that contains the 
+        mutation testing results to be displayed.
+
+        ```html
+        <body>
+        <script src="https://www.unpkg.com/mutation-testing-elements@1.0.2/dist/mutation-test-elements.js"></script>
+        <mutation-test-report-app src="mutation-report.json">
+            Your browser does not support custom elements. Please use a modern browser.
+        </mutation-test-report-app>
+        </body>
+        ``` 
+
+        Instead of loading the report data from a source, it also supports direct property binding:
+
+        ```js
+        document.querySelector('mutation-test-report-app').report = {
+            /*... report data ...*/
+        }
+        ```
+
+        The report json should adhere to the [mutation testing report json schema](https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/mutation-testing-report-schema#readme), 
+        we've designed it with this specific goal in mind.
+
+        For more information on how to use it, as well as where to find the JavaScript implementation, please see the [readme of mutation testing elements](https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/mutation-testing-elements#readme)
+        on github. Want to contribute or suggest changes? Feel free to do so in the [mutation testing elements mono repo](https://github.com/stryker-mutator/mutation-testing-elements).
+
+        ## 🔮 What's next?
+
+        If you've read the introduction of this article, you might have noticed that mutation testing elements 
+        is designed for all mutation testing frameworks, not just Stryker. 
+        In the near future, you might see [mutation testing elements coming to a mutation testing framework near you](https://github.com/infection/infection/issues/340#issuecomment-477148001).
+        If your mutation testing framework isn't supported, why not open up an issue at their side.
+        Spread the word 💖 to help us grow 🌳.
+        
```

Comment:
It'd be awesome to also have some sort of "call to action" to other mutation testing framework developers where we invite them to implement our report and think about improvements, new features, other usages, etc

Reply:
@borNfreee would it be OK if we explicitly name infectionPHP in this blog article? It would help for our call to action 💪

Reply:
Sure!  Thank you @nicojs